tests/markdown/MarkdownBearTest.py

```diff
@@ -29,3 +37,21 @@
     valid_files=(test_file2,),
     invalid_files=(test_file3,),
     settings={'max_line_length': 10})
+
+
+@generate_skip_decorator(MarkdownBear)
+class MarkdownBearMaxLineLengthMessageTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.uut = MarkdownBear(self.section, Queue())
+
+    def test_invalid_message(self):
+        content = test_file3.splitlines()
+        self.section.append(Setting('max_line_length', '10'))
+        with prepare_file(content, None) as (file, fname):
+            with execute_bear(self.uut, fname, file) as results:
+                self.assertEqual(results[0].message,
+                                 'Line must be at most 10 characters'
+                                 '  maximum-line-length  remark-lint')
```

Comment:
hm the spacing is a bit weird, why is this `'  maximum-line-length  remark-lint'` included in the message anyway?